ui/src/components/Notify/UserMessage.module.css

```diff
@@ -0,0 +1,42 @@
+.messageBody {
+  position: flex;
+  justify-content: flex-end;
+  flex-direction: column-reverse;
+  background-color: rgba(255, 255, 255, 0);
+  display: 'flex';
+  z-index: 1000;
+}
+
+.message {
+  position: relative;
+  padding: 0.4em;
+  border-color: #333;
+  opacity: 1;
+}
+
+.INFO {
+  border-top: 5px solid #4e677b;
+  background-color: rgba(91, 141, 180, 0.8);
+  border-radius: 2px;
+}
+
+.WARNING {
+  border-top: 5px solid #2196f3;
+  background-color: rgba(240, 173, 78, 0.8);
+}
+
+.ERROR {
+  border-top: 5px solid #d9534f;
+  background-color: rgba(217, 83, 79, 0.8);
+}
+
+.DEBUG {
+  border-top: 5px solid #d9534f;
+  background-color: rgba(217, 83, 79, 0.8);
+}
+
+.messageText {
```

Comment:
Your classes are namespaced so, technically, nothing forces you to repeat the `message` prefix. `.text` would work fine.

Reply:
This file has been renamed.  not forces at all :innocent: . 
even odd the className could look like, don't care that much  :rofl: